adapters/gumgum/gumgum.go

```diff
@@ -98,12 +109,20 @@ func (g *GumGumAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalRe
 
 	for _, sb := range bidResp.SeatBid {
 		for i := range sb.Bid {
+			// TODO: have gumgum server send mediaType in ext
```

Comment:
Is this something you want to do before merging this PR?

Reply:
I'd like to but unfortunately the change would take longer than this week to get out :( I plan on creating another PR once we are able to send the mediaType in the ext object

Reply:
Not a problem. The TODO makes it look like something is missing from the adapter. Since the adapter as complete as possible right now (until you add server side support), could you please remove the TODO?

Reply:
@SyntaxNode - removed the comment! Is there anything else that needs to be modified?